src/aasthaJJun20/ItemIDFromURL.java

```diff
@@ -0,0 +1,53 @@
+package aasthaJJun20;
+/*Assignment 33 : 22nd July 2020
+
+Extract ItemId from below input
+
+String input = https://www.facebook.com/technocredits/itemId=12345/
+String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+String input = https://www.facebook.com/technocredits/empid=123/
+
+Output : 
+item id is 12345 
+        OR
+no item id present.*/
+public class ItemIDFromURL {
+	void findItemIDFromURL(String url) {
+		System.out.println("\nString input : " + url);
+		String temp = "";
+		try {
+			temp = url.substring(url.indexOf("itemId"));
+		} catch (StringIndexOutOfBoundsException stringIndexOutOfBoundsException) {
+			System.out.println("No item id present");
+			return;
+		}
+		String itemId = "";
+		for (int index = 0; index < temp.length(); index++) {
```

Comment:
if (input.contains("itemId=")) {
			String str = input.split("itemId=")[1];
			String output = "";
			for (int index = 0; index < str.length(); index++) {
				if (Character.isDigit(str.charAt(index))) {
					output += str.charAt(index);
				} else {
					break;
				}
			}
			System.out.println("Item id is " + output);
		} else {
			System.out.println("No itemID present.");
		}

Reply:
Thanks Krishna!!